flink-tests/src/test/java/org/apache/flink/runtime/util/ExceptionUtilsITCases.java

```diff
@@ -42,31 +45,83 @@
  * Tests for {@link  ExceptionUtils} which require to spawn JVM process and set JVM memory args.
  */
 public class ExceptionUtilsITCases {
+	private static final int DIRECT_MEMORY_SIZE = 1 << 20; // 128Mb
```

Comment:
Can we make this size smaller?

Reply:
The comment is wrong, it is actually 1Mb. I can make it even 10Kb but it does not really affect the runtime too much, at least locally.